src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
 export interface Figure {
```

Comment:
This interface should dictate shape of the blow classes, according to the requirements:
<img width="859" alt="image" src="https://github.com/mate-academy/ts_figures_classes/assets/87986955/f520f8fa-57a2-4933-8630-c687a92f5c52">
- add missing properties

Reply:
Also the getArea is wrongly typed, it doesn't return number or Error, it always returns a number